packages/opencensus-core/test/test-view.ts

```diff
@@ -296,6 +297,70 @@ describe('BaseView', () => {
       });
     });
 
+    describe(
+        'DISTRIBUTION aggregation type: record with measurements in succession from a single view and single measure',
+        () => {
+          const view: View = new BaseView(
+              'test/view/name', measure, AggregationType.DISTRIBUTION,
+              ['testKey1', 'testKey2'], 'description test', buckets);
+          let total = 0;
+          for (const value of measurementValues) {
+            total += value;
+            const measurement = {measure, tags, value};
+            const measurement1 = {measure, tags: tags1, value};
+            view.recordMeasurement(measurement);
+            view.recordMeasurement(measurement1);
+          }
+
+          it('should has points', () => {
```

Comment:
nit: "should have points"

Reply:
Done. Corrected typos in other existing tests.